src/Extracting/Strategies/Responses/ResponseCalls.php

```diff
@@ -90,7 +90,11 @@ public function makeResponseCall(Route $route, array $context, array $rulesToApp
      */
     private function configureEnvironment(array $rulesToApply)
     {
-        $this->startDbTransaction();
+        // Start transactions for all connections since we don't know wich one is used
+        $connections = array_keys(config('database.connections'));
```

Comment:
Can we rather move this whole logic into the startDbTransactions method? Same for the endDbTransactions.The configureEnvironment method should just be a high-level list of steps.

Also, don't forget to wrap each individual call to app('db')->... in a try/catch.

Reply:
Done!